azurerm/resource_arm_subnet.go

```diff
@@ -175,6 +181,21 @@ func resourceArmSubnetCreateUpdate(d *schema.ResourceData, meta interface{}) err
 		AddressPrefix: &addressPrefix,
 	}
 
+	if v, ok := d.GetOk("disable_private_link_service_network_policies"); ok {
+		// This is strange logic, but to get the schema to make sense for the end user
+		// I exposed it with the same name that the Azure CLI does to be consistent
+		// between the tool sets, which means true == Disabled.
+		//
+		// To enable private endpoints you must disable the network policies for the
+		// subnet because Network policies like network security groups are not
+		// supported by private endpoints.
+		p := "Enabled"
```

Comment:
What is the default behaviour? is it Enabled by default then?

What does the portal do?

Reply:
Yes, by default it is enabled. In the portal if you provision a service or endpoint it will change the subnet to disabled.